### PR TITLE
Next batch of fixes

### PR DESCRIPTION
**Pull request by AlexIljin**
Hi!

  I have a couple of documentation updates here, and a couple of changes to the PDF vocab.

### CODE REVIEW

File: basis/concurrency/distributed/distributed-docs.factor [hunk 1]
```diff
@@ -39,9 +41,9 @@ $nl
 $nl
 "It is also possible to use " { $link send-synchronous } " to receive a "
 "response to a distributed message. When an instance of " { $link thread } " "
-"is marshalled it is converted into an instance of " { $link remote-thread }
+"is marshalled, it is converted into an instance of " { $link remote-thread }
 ". The receiver of this can use it as the target of a " { $link send }
-" or " { $link reply } " call." ;
+", " { $link send-synchronous } " or " { $link reply-synchronous } " call." ;
```
Comment (mrjbq7) [inline basis/concurrency/distributed/distributed-docs.factor]: Why did you remove ``send`` and ``reply`` in favor of the synchronous versions?  Maybe we should mention both versions.

Comment (AlexIljin) [inline basis/concurrency/distributed/distributed-docs.factor]: Ain't no such thing as `reply`, my friend.
Also, I didn't remove `send`.

File: extra/pdf/layout/layout.factor [hunk 2]
```diff
@@ -359,7 +359,7 @@ TUPLE: pdf-info title timestamp producer author creator ;
     pdf-info new
         now >>timestamp
         "Factor" >>producer
-        "USER" os-env "unknown" or >>author
+        "USER" os-env \ developer-name get-global "unknown" or or >>author
```
Comment (mrjbq7) [inline extra/pdf/layout/layout.factor]: I wonder if we should instead have symbols that are for the pdf-author, instead of trying to guess this way.

```factor
SYMBOLS: pdf-producer pdf-author pdf-creator ;
```

Comment (AlexIljin) [inline extra/pdf/layout/layout.factor]: Good idea.

File: basis/concurrency/distributed/distributed-docs.factor (hunk 1 above)
Comment (mrjbq7) [inline basis/concurrency/distributed/distributed-docs.factor]: Ahh, I missed that.

File: extra/pdf/layout/layout.factor (hunk 2 above)
Comment (mrjbq7) [inline extra/pdf/layout/layout.factor]: Want to make that change? Or should I pull these commits and then make it myself?

Comment (AlexIljin) [inline extra/pdf/layout/layout.factor]: OK, I'll do it.

Comment (AlexIljin) [inline extra/pdf/layout/layout.factor]: Done!

**Comment by mrjbq7** [comment]
Merged!
